### PR TITLE
COMMERCE-1471 card restyled on bom

### PR DESCRIPTION
**Pull request by fabio-d-m**

### CODE REVIEW

**Comment by ethanbustad** [comment]
ci:test

**Comment by fabio-d-m** [comment]
@ethanbustad rebased and pushed

**Comment by ethanbustad** [comment]
ci:test

File: commerce-theme-minium/commerce-theme-minium/src/css/minium/components/bom.scss [hunk 1]
```diff
@@ -314,4 +292,42 @@
...
+	.card-body {
+		padding: 20px;
+	}
+
+	&.image-card .bom-card__folder-picture-container{
+		background-color: #E7EEFE;
+		background-image: none;
+	}
+
+	.card-title {
```
Comment (ethanbustad) [inline commerce-theme-minium/commerce-theme-minium/src/css/minium/components/bom.scss:321]: Hey Fabio, these selectors should be sorted (assuming that doesn't break the cascade).

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by marco-leo** [comment]
@FabioDiegoMastrorilli can you please fix this pull?

**Comment by ethanbustad** [comment]
ci:test
tests should be fixed in upstream now

File: commerce-theme-minium/commerce-theme-minium/src/css/minium/components/bom.scss (hunk 1 above)
Comment (ethanbustad) [reply commerce-theme-minium/commerce-theme-minium/src/css/minium/components/bom.scss:321]: @FabioDiegoMastrorilli it looks like this still needs to be sorted -- can you send a follow-up PR?

**Pull request merged**